metabase/metabase
# feat(sdk): revamp CreateQuestion and create question behaviour

Closes https://github.com/metabase/metabase/issues/50082

### Description

This PR updates the underlying implementation of the `CreateQuestion` component to use `InteractiveQuestion` directly, and alters the default behaviour for saving questions.

### How to verify

- Use the `<CreateQuestion />` component
- There should be the "Show Visualization / Show Editor" button instead of tabs
- When saving the question, the question title should update. It should NOT show "Question not found"
- If you make a change to the saved question and click "Save" again, it should have the "- Modified" suffix

### Demo

[Private Loom](https://loom.com/share/aafc92798b814f56bd989765153920dc)

How it looks in Shoppy

![CleanShot 2567-11-20 at 02 58 41@2x](https://github.com/user-attachments/assets/a65690f9-c335-4bb8-a5ae-f7dfa5bfbf0b)

### Checklist
